Main/lib/sql/Postgres/createVdiControlTables.sql

```diff
@@ -83,40 +76,139 @@ CREATE TABLE VDI_CONTROL_:VAR1.dataset_dependency (
+    study_design TEXT,
+    study_type VARCHAR(500),
+    participant_ages VARCHAR(500),
+    sample_year_start SMALLINT,
+    sample_year_end SMALLINT,
+    CONSTRAINT valid_year_range CHECK (
+        (sample_year_start IS NULL AND sample_year_end IS NULL) OR 
+        (sample_year_start IS NOT NULL AND sample_year_end IS NOT NULL AND sample_year_start <= sample_year_end)
+    ),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE INDEX VDI_CONTROL_:VAR1.idx_dataset_characteristics ON dataset_characteristics(dataset_id);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_country (
+    dataset_id INTEGER NOT NULL,
+    country VARCHAR(255) NOT NULL,
+    PRIMARY KEY (dataset_id, country),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_species (
+    dataset_id INTEGER NOT NULL,
+    species VARCHAR(500) NOT NULL,
+    PRIMARY KEY (dataset_id, species),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_disease (
+    dataset_id INTEGER NOT NULL,
+    disease VARCHAR(500) NOT NULL,
+    PRIMARY KEY (dataset_id, disease),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_associated_factor (
+    dataset_id INTEGER NOT NULL,
+    factor VARCHAR(500) NOT NULL,
+    PRIMARY KEY (dataset_id, factor),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_sample_type (
+    dataset_id INTEGER NOT NULL,
+    type VARCHAR(500) NOT NULL,
+    PRIMARY KEY (dataset_id, type),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_doi (
+    dataset_id VARCHAR(255) NOT NULL,
+    doi VARCHAR(500) NOT NULL,
+    description TEXT,
+    PRIMARY KEY (dataset_id, doi),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_bioproject_id (
+    dataset_id VARCHAR(255) NOT NULL,
+    bioproject_id VARCHAR(255) NOT NULL,
+    description TEXT,
+    PRIMARY KEY (dataset_id, bioproject_id),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+);
+
+CREATE TABLE VDI_CONTROL_:VAR1.dataset_link (
+    dataset_id VARCHAR(255) NOT NULL,
+    dataset_uri VARCHAR(2048) NOT NULL,
+    shares_records BOOLEAN NOT NULL DEFAULT FALSE,
+    PRIMARY KEY (dataset_id, dataset_uri),
+    FOREIGN KEY (dataset_id) REFERENCES VDI_CONTROL_:VAR1.dataset(dataset_id)
+    );
+
+CREATE INDEX idx_linked_datasets_dataset ON linked_datasets(dataset_id);
+
 -- convenience view showing datasets visible to a user that are fully installed, and not deleted
 -- application code should use this view to find datasets a user can use
 CREATE VIEW VDI_CONTROL_:VAR1.AvailableUserDatasets AS
```

Comment:
View still contains references to `short_name`, `short_attribution`, and `category`